app/controllers/api/service_templates_controller.rb

```diff
@@ -52,5 +53,9 @@ def unarchive_resource(type, id, _data)
     def set_additional_attributes
       @additional_attributes = %w(config_info)
     end
+
+    def decode_picture(data)
+      data["picture"]["content"] = Base64.strict_decode64(data["picture"]["content"])
```

Comment:
Can Picture.create_from_base64(...) be used here ?

Reply:
there are three ways to do this: the UI, the options hash, and this call. the only place we have to worry about the encoding of the content is here. i think core should handle the picture create, and that side is working fine as long as you have this change.

Reply:
but actually anyway, it can't for a better reason, doing so runs into this fun: `ActiveRecord::RecordNotSaved: You cannot call create unless the parent is saved`

Reply:
If we were strictly REST here, I think the picture upload would be an independent call to upload the image then `data` in this call would include `"picture" => /api/pictures/:id"`, not the raw content.  But I think that change is out of scope for this PR.

Reply:
👍 Thanks.